packages/data-explorer/src/components/SearchComponent.vue

```diff
@@ -50,12 +50,13 @@ export default {
   methods: {
```

Comment:
why does the search component need to be updated when pagination is added ?

Reply:
Reverted changes. Doesn't seem to do much.